data/ideas.json

```
@@ -184,5 +184,11 @@
         "description": "An app or website to generate a possible, safest and most acceptable roadmap for the query user seraches about like full stack developement, software developement etc. and also shows the details of each step to gain success",
         "level": "advanced",
         "tags": "roadmap, success, process"
+    },
+    {
+        "title": "Book App",
+        "description": "A cataloging web application where users can search for books by title or author using an API Eg: Rapid API.",
+        "level": "advanced",
+        "tags": "javascript, api, es6"
```

- The first tag is used to match projects with an idea, so please add the `book-app` tag
- Remove the programming tag, due to is only an idea so the everyone could make on whatever programming language
```suggestion
        "tags": "book-app"
```

---

Shouldn't it book API, should?